heron/proto/scheduler.proto

```diff
@@ -28,6 +29,11 @@ message RestartTopologyRequest {
   required int32 container_index = 2;
 }
 
+message UpdateTopologyRequest {
+  required heron.proto.system.PackingPlan currentPackingPlan = 1;
+  required heron.proto.system.PackingPlan proposedPackingPlan = 2;
+}
+
```

Comment:
If this is indeed a request - do we need a response? In general, for control messages, it will always good to send a response back.